client/src/components/Tool/ToolCard.vue

```diff
@@ -77,10 +79,16 @@ const props = defineProps({
         type: Boolean,
         default: false,
     },
+    editorView: {
```

Comment:
This should be renamed to something meaningful, like maybe `allow-editing-credentials`.